frontends/web/src/components/devices/bitbox02bootloader/bitbox02bootloader.tsx

```diff
@@ -58,8 +58,14 @@ export const BitBox02Bootloader = ({ deviceID }: TProps) => {
       const value = Math.round(status.progress * 100);
       contents = (
         <div className="box large">
+          <h2 className="subTitle">
+            {t('bb02Bootloader.upgradeTitle', { context: (versionInfo.erased ? 'install' : '') })}
+          </h2>
           { versionInfo.additionalUpgradeFollows ? (
-            <p>{t('bb02Bootloader.additionalUpgradeFollows')}</p>
+            <>
+              <p>{t('bb02Bootloader.additionalUpgradeFollows1')}</p>
+              <p>{t('bb02Bootloader.additionalUpgradeFollows2')}</p>
+            </>
           ) : null }
```

Comment:
Nice, for consistency could you add `className="subTitle"` to the other `h2` element for the welcome message?

```jsx
<h2>{t('welcome.title')}</h2>
```

(here the welcome text  should have the same style as the others)
![Screen Shot 2024-04-10 at 12 26 07](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/a557e2a8-1a2f-4dd9-9b17-fee6d8ef19b6)

![Screen Shot 2024-04-10 at 12 26 16](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/407e8689-5e7e-4664-ab28-cce1f277c121)


![Screen Shot 2024-04-10 at 12 26 38](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/36808dc5-a684-404c-8b41-d95465442359)
![Screen Shot 2024-04-10 at 12 27 31](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/5de67048-aa4a-475a-8338-db0ab6b1d934)